### DISCUSSION TITLE
TV Kanagawa

### DISCUSSION THREAD

**Discussion by Atekuro95** [iptv-org/community#645] [Q&A]
**Country of the channel: Japan**

**Channel Name: TV Kanagawa**

**Channel ID: JOKMDTV.jp**

**Channel Language: Japanese**

**Additional Information (optional):**

**Comment by BellezaEmporium** [comment]
The only kind of link I've found is this : https://www.youtube.com/channel/UCtDEkfJlNHq_ONjQnsX5YeA

**Comment by BellezaEmporium** [comment]
with programmation here ; https://www.tvk-yokohama.com/bangumi/ they work with a 28-hour system, but understand that after the 24hr mark is past midnight and you have programs going until 2:30 am, and going back at 5 am.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19648
